crates/any/any-counterparty/Cargo.toml

```diff
@@ -1,26 +1,27 @@
 [package]
 name = "hermes-any-counterparty"
-version      = { workspace = true }
-edition      = { workspace = true }
-license      = { workspace = true }
-keywords     = { workspace = true }
-repository   = { workspace = true }
-authors      = { workspace = true }
+version = { workspace = true }
+edition = { workspace = true }
+license = { workspace = true }
+keywords = { workspace = true }
+repository = { workspace = true }
+authors = { workspace = true }
 rust-version = { workspace = true }
-description  = """
+description = """
     Implementation of an IBC Relayer in Rust, as a library
 """
 
 [dependencies]
-ibc-proto                               = { workspace = true }
-tendermint-proto                        = { workspace = true }
-hermes-error                            = { workspace = true }
-hermes-relayer-components               = { workspace = true }
-hermes-cosmos-chain-components          = { workspace = true }
-hermes-cosmos-chain-preset              = { workspace = true }
-hermes-encoding-components              = { workspace = true }
-hermes-protobuf-encoding-components     = { workspace = true }
+ibc-proto                           = { workspace = true }
+tendermint-proto                    = { workspace = true }
+hermes-error                        = { workspace = true }
+hermes-relayer-components           = { workspace = true }
+hermes-core                         = { workspace = true }
```

Comment:
I'm not sure I understand the purpose of `hermes-core` here.
Wouldn't it allow to reduce the number of dependencies?

Reply:
Yup, it is a meta-crate to reduce the number of dependencies, so that high level crates can include just one crate instead of ten. Many popular crates like tokio are in fact meta crates that just re-export other smaller crates.

Reply:
But those this mean we can remove the `hermes-encoding-components` and use `hermes_core::encoding_components` instead?

Reply:
Yeah we can. I could miss some of the renaming.

Reply:
I see, I commented before seeing the cli crates which use the new renaming/dependencies